src/Adapters/MessageQueue/EventEmitterMQ.js

```diff
@@ -0,0 +1,65 @@
+import events from 'events';
+
+let emitter = new events.EventEmitter();
```

Comment:
oh man, i'm getting confused on local emitter, this.emitter and this files scope emitter.....